src/main/java/uk/gov/hmcts/reform/et/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -73,10 +76,49 @@ public static Boolean isHearingIsLinkedFlag(CaseData caseData) {
         return Boolean.parseBoolean(caseData.getHearingIsLinkedFlag());
     }
 
-    public static String getTribunalAndOfficeLocation(CaseData caseData) {
-        return caseData.getTribunalAndOfficeLocation();
+    public static String getTribunalAndOfficeLocation() {
+        return null;
```

Comment:
Why is this null?

Reply:
It's a fix from https://tools.hmcts.net/jira/browse/RET-3992 that I incorporated into this ticket, Ben confirmed it can be set to null